terraform/031-email-service/main.tf

```diff
@@ -50,6 +50,52 @@ resource "random_id" "access_token_idsync" {
   byte_length = 16
 }
 
+/*
+ * Create role for access to SES
+ */
+resource "aws_iam_role" "ses" {
+  name = "ses-${var.idp_name}-${var.app_name}-${var.app_env}-${var.aws_region}"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Sid    = "ECSAssumeRoleSES"
+        Effect = "Allow"
+        Principal = {
+          Service = [
+            "ses.amazonaws.com",
+            "ecs-tasks.amazonaws.com",
+          ]
+        }
+        Action = "sts:AssumeRole"
+      }
+    ]
```

Comment:
I'm curious: how did you learn/find what needed to be in this policy?

Reply:
I did it 2+ years ago, so I don't remember. I did try to relearn it recently, though, which is why I included a link to the AWS documentation.

Reply:
Oh, I guess the AWS link is for the other policy. I think we've used a policy similar to this one in other places. Can't remember where, at the moment.